aws/api.ts

```diff
@@ -293,7 +296,15 @@ function convertHandlers(handlers: cloud.RouteHandler[]): lambda.Callback<x.Requ
         next();
     };
 
-    return result;
+    const routeName = name + sha1hash(route.method + ":" + route.path);
+
+    // Create the CallbackFunction in the cloud layer as opposed to just passing the callback down
+    // to pulumi-aws.  This ensures that the right configuration values are used that will
+    // appropriately respect user settings around things like codepaths/policies etc.
```

Comment:
Not sure I follow this comment. Maybe I've just lost track of the changes around here, but how does this respect user settings (and what does that mean)?